From 8a64bb4e3bb6c9a41e140d6e7c2c450cb0d8ec50 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 11 Mar 2015 14:17:52 +0100
Subject: [PATCH] Set system time zone in test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The 01conversions.t fails since Date-Manip-6.49 because it takes time
zone from syste. Whether it's a bug or a feature, setting TZ
enviroment variable makes the test happy.

CPAN RT#102670

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 t/01conversions.t | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/t/01conversions.t b/t/01conversions.t
index f4b308c..9f51b61 100644
--- a/t/01conversions.t
+++ b/t/01conversions.t
@@ -2,6 +2,11 @@ use strict;
 
 use Test::More;
 
+## Set the system time zone because of CPAN RT#102670
+BEGIN {
+    $ENV{TZ}="US/Eastern";
+}
+
 use DateTime::Format::DateManip;
 use DateTime;
 use Date::Manip;
-- 
2.1.0

